### PR TITLE
Adding support for hashmap for the case of json value without a schema

### PR DESCRIPTION
**Pull request by Uriende**
Relates to issue #48 
Added support for hashmap types.

Let me know if something else needs to be added or changed.

### CODE REVIEW

File: src/main/java/io/aiven/kafka/connect/http/converter/RecordValueConverter.java [hunk 1]
```diff
@@ -36,7 +40,8 @@
     public String convert(final SinkRecord record) {
         if (!converters.containsKey(record.value().getClass())) {
             throw new DataException(
-                    "Record value must be String or Schema Struct, but " + record.value().getClass() + " is given");
+                    "Record value must be String or Schema Struct or HashMap,"
```
Comment (jjaakola-aiven) [inline src/main/java/io/aiven/kafka/connect/http/converter/RecordValueConverter.java:43]:
```suggestion
                    "Record value must be String, Schema Struct or HashMap," 
```

**Review by jjaakola-aiven — CHANGES_REQUESTED** [review]
@Uriende thank you for the contribution.

The change is good, if the `[key|value].converter.schemas.enable=false` and `org.apache.kafka.connect.json.JsonConverter` is used a HashMap is passed to record converter.

What I'd like to ask in addition is an integration test for reproduction and squashing to single commit.

**Comment by Uriende** [comment]
@jjaakola-aiven Added a specific integration test for json and squashed. I've considered also testing json with a schema but the integration tests are using an older version of the confluent platform that doesn't yet support json with schema to my understanding.

File: src/integration-test/java/io/aiven/kafka/connect/http/JsonIntegrationTest.java [hunk 2]
```diff
@@ -0,0 +1,187 @@
...
+
+    private static File pluginsDir;
+
+    @Container
+    private final KafkaContainer kafka = new KafkaContainer()
+            .withNetwork(Network.newNetwork())
+            .withEnv("KAFKA_AUTO_CREATE_TOPICS_ENABLE", "false");
+
+    @Container
+    private final SchemaRegistryContainer schemaRegistry = new SchemaRegistryContainer(kafka);
```
Comment (jjaakola-aiven) [inline src/integration-test/java/io/aiven/kafka/connect/http/JsonIntegrationTest.java:53]: SchemaRegistryContainer is not used in this test and can be removed.

File: src/integration-test/java/io/aiven/kafka/connect/http/JsonIntegrationTest.java [hunk 3]
```diff
@@ -0,0 +1,187 @@
...
+import io.aiven.kafka.connect.http.mockserver.BodyRecorderHandler;
+import io.aiven.kafka.connect.http.mockserver.MockServer;
+import org.apache.kafka.clients.admin.AdminClient;
+import org.apache.kafka.clients.admin.AdminClientConfig;
+import org.apache.kafka.clients.admin.NewTopic;
+import org.apache.kafka.clients.producer.KafkaProducer;
+import org.apache.kafka.clients.producer.ProducerConfig;
+import org.apache.kafka.clients.producer.ProducerRecord;
+import org.apache.kafka.clients.producer.RecordMetadata;
+import org.junit.jupiter.api.*;
```
Comment (jjaakola-aiven) [inline src/integration-test/java/io/aiven/kafka/connect/http/JsonIntegrationTest.java:13]: Expand wildcard imports.

**Review by jjaakola-aiven — CHANGES_REQUESTED** [review]
Thank you. There are two items to clean, wildcard imports and unnecessary schema container setup.

**Comment by Uriende** [comment]
Cleaned the imports and remvoed the unused container. Also added to copyright comment that I noticed was also missing from my file

**Comment by Uriende** [comment]
Saw the issues. Sorry forgot to run the build locally. I'll fix everything now.

**Review by jjaakola-aiven — APPROVED** [review]

**Pull request merged**